sklearn/neural_network/tests/test_mlp.py

```diff
@@ -495,6 +495,40 @@ def test_predict_proba_multilabel():
     assert_array_equal(y_log_proba, np.log(y_proba))
 
 
+def test_shuffle():
+    X, y = make_multilabel_classification(n_samples=200, n_features=10,
+                                          n_classes=5, n_labels=2,
+                                          allow_unlabeled=False,
+                                          random_state=0)
+
+    # Ensure shuffling happens or doesn't happen when passed as an argument
```

Comment:
"ensure shuffling parameter has some effect" is all you are testing

Reply:
This is fair, the test could be more direct. The most direct effect of shuffle is the resulting coefficients so I rewrote the test to look at them instead.

I can’t really think of a better way to test it than that, but I made sure this new test consistently works across different sample sizes and random states so it is a genuine test of the shuffle behavior.

Reply:
I didn't really have a problem with the test... I had a problem with the comment.